contrib/pgn-extract/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "pgn-extract"
+pkgver = "22.11"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_check_target = "all"
```

Comment:
pass make_build_args = ["OPTIMISE="] too  

also add `$(CFLAGS)` to

```makefile
pgn-extract : $(OBJS)
	$(CC) $(DEBUGINFO) $(ORIGCFLAGS) $(CPPFLAGS) $(LDFLAGS) $(OBJS) $(LIBS) -o pgn-extract
```

Reply:
If i just patch $(CFLAGS) here it breaks. I guess we only need cflags from cports

Reply:
ah, sure, cause `-c` is in them